src/v3/src/components/AuthCoin/AuthCoin.tsx

```diff
@@ -44,8 +46,8 @@ const AuthCoin: FunctionComponent<AuthCoinProps> = (props) => {
           alt={authCoinConfigByAuthKey.description}
           className="custom-logo"
           sx={{
-            width: (theme) => theme.spacing(6),
-            height: (theme) => theme.spacing(6),
+            width: () => Tokens.Spacing7,
+            height: () => Tokens.Spacing7,
```

Comment:
OD 0.x spacing values are mapped to one value larger in OD 1.x because Odyssey decided that `Spacing0` should actually map to 0 rather than a non-0 number so the range was shifted up by 1

Reply:
Can we just use
```
width: Tokens.Spacing7,
height: Tokens.Spacing7,
```
?

Reply:
Yep, good catch. I think there's a lot of other cases where I left the arrow function, will update